### PR TITLE
Some advancements  

### PR DESCRIPTION
**Pull request by JiroCab**
# 7 new Advancements 
image
 - +4 related to crafting saplings from different biomes base of git hub wiki ( #344 )
 - +1 goal for collecting all the saplings
 - +1 for getting salt
 - +1 getting cinnamon
 
 # internal changes
 - Seed related 'gather_x' has been renamed to 'gather_seed_x' (will lose progress, im sorry for not naming properly at first)
 - Fixed some grammatical inconsistency
 - Removed tag 'advancements_hoe' , root now use "minecraft:hoe" for better compatibility
 - Updated tag "advancements_food_big" & "advancements_food_crafted" with the new items
 
 _Im sorry translators_

### CODE REVIEW

**Comment by JiroCab** [comment]
Thanks for the feed back and made changes accordantly 

_Hopefully this wont break things as much _

File: shared/src/main/resources/assets/croptopia/lang/de_de.json [hunk 1]
```diff
@@ -458,6 +458,62 @@
   "block.croptopia.cinnamon_wood": "Zimtholz",
   "block.croptopia.stripped_cinnamon_wood": "Entrindetes Zimtholz",
 
+  "advancements.croptopia.root.description": " 'You Won't Get Anything Done Hoeing Like That' ",
```
Comment (Ayutac) [inline shared/src/main/resources/assets/croptopia/lang/de_de.json:461]: Why the space before and after the '?

File: shared/src/main/resources/assets/croptopia/lang/en_us.json [hunk 2]
```diff
@@ -567,6 +567,23 @@
   "advancements.croptopia.gather_food.description": "Eat everything there is too eat",
   "advancements.croptopia.knife.title": "Iron Torch",
   "advancements.croptopia.knife.description": "Make a Knife",
+  "advancements.croptopia.tofu.title": "Meaty beans",
+  "advancements.croptopia.tofu.description": "Make Tofu",
+  "advancements.croptopia.cinnamon.title": "Oddly tasty sticks",
+  "advancements.croptopia.cinnamon.description": "Strip a cinnamon tree from your local jungle",
+  "advancements.croptopia.salt.title": "Sadly, wont give more Fps",
```
Comment (Ayutac) [inline shared/src/main/resources/assets/croptopia/lang/en_us.json:574]: FPS should be all capitalized

**Review by Ayutac — APPROVED** [review]
Your Advancement capitalization still differs from the example set, but imo thats a problem for Thonk :P

Now stuff won't get broken and I am happy. In fact I didn't mean to ADD the new advancements to the other lang files, but to change the stuff you changed in the en_us version. But undoing these changes with the internal names is the better alternative anyway.

**Comment by ExcessiveAmountsOfZombies** [comment]
thank you for the updated advancements 😄

**Pull request merged**